e2e/max_flow_test/test_paths/test.yml

```diff
@@ -0,0 +1,8 @@
+query: >
+  MATCH (source {id: 0}), (sink {id: 5})
+  CALL max_flow.get_paths(source, sink)
+  YIELD path, flow
+  RETURN count(path) AS count
+
+output:
+    - count: 4
```

Comment:
Create another empty test case, and test case with multiple components, since that might be the edge-case